src/lib/views/account-settings/UpdatePassword.svelte

```diff
@@ -0,0 +1,319 @@
+<script lang="ts">
+  /* This Source Code Form is subject to the terms of the Mozilla Public
+   * License, v. 2.0. If a copy of the MPL was not distributed with this
+   * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+  import { createEventDispatcher } from "svelte";
+  import { getContext } from "svelte";
+  import type { NotificationStore } from "$lib/components/notifications";
+  import type { AppStore } from "$lib/stores/types";
+  import Button from "../../../lib/components/Button.svelte";
+  import "../../../lib/components/auth-cards/Auth.css";
+  const dispatch = createEventDispatcher();
+  const store: AppStore = getContext("rally:store");
+  const notifications: NotificationStore = getContext("rally:notifications");
+
+  let btnDisabled = true;
+  let oldPasswordEl;
+  let newPasswordEl;
+  let newPasswordEl2;
+  let capital;
+  let number;
+  let length;
+  let letter;
+  let passwordVisible1 = false;
+  let passwordVisible2 = false;
+  let passwordVisible3 = false;
+  const minPasswordLength = 8;
+  let pattern = "(?=.*d)(?=.*[a-z])(?=.*[A-Z]).{8,}";
+
+  let pwErr = false;
```

Comment:
for naming consistency, consider spelling out "password" for these as you have in other instances.

Reply:
Definitely